src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -8,18 +8,19 @@
 
 AUTOREST     = "../../core/AutoRest/bin/Debug/net451/win7-x64/AutoRest.exe"
 
-SWAGGER_VERSIONS = {
+ARM_SWAGGERS = {
+	# asazure: {version: "2016-05-06"},
 	authorization: {version: "2015-07-01"},
 	batch: {version: "2015-12-01", swagger: "BatchManagement"},
 	cdn: {version: "2016-04-02"},
 	cognitiveservices: {version: "2016-02-01-preview"},
-	compute: {version: "2016-03-30"},
+	compute: {version: "2016-03-30"}, #composite swagger (includes container service)
 	# containerservice: {version: "2016-03-30", swagger: "containerservice"},
 	# commerce: {version: "2015-06-01-preview"},
 	datalake_analytics: {
 		account: {version: "2015-10-01-preview"},
 		# catalog: {version: "2016-06-01-preview"},
-		job:  {version: "2016-03-20-preview"}
+		# job:  {version: "2016-03-20-preview"}
```

Comment:
If I try to generate this one, the result is a bit ugly. And we are not generating job right now. In the other hand, these are dataplane.